src/crawler.ts

```diff
@@ -200,11 +308,16 @@ export class Crawler {
 
     if (this.params.screencastPort) {
       transport = new WSTransport(this.params.screencastPort);
-      logger.debug(`Screencast server started on: ${this.params.screencastPort}`, {}, "screencast");
-    } else if (this.params.redisStoreUrl && this.params.screencastRedis) {
-      transport = new RedisPubSubTransport(this.params.redisStoreUrl, this.crawlId);
-      logger.debug("Screencast enabled via redis pubsub", {}, "screencast");
+      logger.debug(
+        `Screencast server started on: ${this.params.screencastPort}`,
+        {},
+        "screencast"
+      );
     }
+    // } else if (this.params.redisStoreUrl && this.params.screencastRedis) {
+    //   transport = new RedisPubSubTransport(this.params.redisStoreUrl, this.crawlId);
+    //   logger.debug("Screencast enabled via redis pubsub", {}, "screencast");
+    // }
```

Comment:
Curious why this is commented out - are we removing this option or is there a follow-up? Option still seems available in argParser

Reply:
Yeah, removing this - it hasn't been used in a long time, and proved to be inefficient compared to direct WS transport, should remove in a follow-up